### PR TITLE
Chore: add license

### PR DESCRIPTION
**Pull request by katspaugh**
Copied the license from safe-react.

### CODE REVIEW

**Review by usame-algan — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# 💝 Code Style Guidelines
+
+## Principles
```
Comment (iamacook) [inline CONTRIBUTING.md:3]: Should we add something about tests here?

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018-2022 Safe Ecosystem Foundation
```
Comment (iamacook) [inline LICENSE.md:3]: Is this correct seeing as the foundation is new, or should it reference Gnosis first?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- Same with components: don’t write giant components, write small composable components
+- Prefer map/filter over reduce/forEach
+- Watch out when using destructive methods like `pop` or `sort` (yes, sort is destructive!)
+- Avoid initializing things on the module level, prefer to export an init function instead
+
+## Reactive programming
+
+- Keep in mind the React component life-cycle, avoid excessive re-renders
+- Glue regular JS functions and events with React using hooks
+- Write small useEffect hooks that do just one thing and have only necessary dependencies
```
Comment (iamacook) [inline CONTRIBUTING.md:26]:
```suggestion
- Write small `useEffect` hooks that do just one thing and have only necessary dependencies
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## Variable/function naming
+
+Infamously, the hardest problem in computer science.
+
+- Components are classes, so their names should be in CapitalCase
```
Comment (iamacook) [inline CONTRIBUTING.md:32]:
```suggestion
- Components are classes, so their names should be in PascalCase
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+- Config-like constants should be in UPPER_CASE, e.g. `INFURA_URL`
+- Regular const variables should be in camelCase
```
Comment (iamacook) [inline CONTRIBUTING.md:34]:
```suggestion
- Regular `const` variables should be in camelCase
```

File: LICENSE.md (hunk 2 above)
Comment (katspaugh) [reply LICENSE.md:3]: It's the license from safe-react, have you read the PR description?

File: CONTRIBUTING.md (hunk 1 above)
Comment (katspaugh) [reply CONTRIBUTING.md:3]: We should definitely add more than just code style there.

Comment (iamacook) [reply CONTRIBUTING.md:3]: A good point of reference is the Notion document we have about code style. Maybe we should discuss it after the standup tomorrow and gather ideas? This'll definitely develop over time though.

File: LICENSE.md (hunk 2 above)
Comment (iamacook) [reply LICENSE.md:3]: It slipped my mind that we adjusted `safe-react` directly to the foundation.

**Review by iamacook — APPROVED** [review]

**Pull request merged**